src/js/components/Tag/propTypes.js

```diff
@@ -1,11 +1,8 @@
 import PropTypes from 'prop-types';
-import { genericProps } from '../../utils/general-prop-types';
 
 let PropType = {};
 if (process.env.NODE_ENV !== 'production') {
   PropType = {
-    ...genericProps,
```

Comment:
Why do we remove genericProps / as here? If so, should we be spreading Box prop types?

Reply:
Since Tag is built on box, it will inherit the props. Resolving this comment.